src/tokenize.cpp

```diff
@@ -11,6 +11,7 @@
 #include "char_table.h"
 #include "prototypes.h"
 #include "chunk_list.h"
+#include <stdbool.h>
```

Comment:
we wanted to have sources which all are "uncrustify-ed" them self with the actual config file.